scripts/node/node.start.sh

```diff
@@ -3,3 +3,4 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 docker build -f Dockerfile.dev . -t summit-frontend-wss
 
 docker run --rm --net=summit -p 3000:3000 -v "$(pwd)/src/:/usr/src/app/src/" -e DATA --name=summit-frontend-wss  summit-frontend-wss
+# docker run --rm --net=infinispan-docker-compose_summit -p 3000:3000 -v "$(pwd)/src/:/usr/src/app/src/" -e DATA --name=summit-frontend-wss  summit-frontend-wss
```

Comment:
the network name needs to change with compose

Reply:
Gotcha, this is OK. Let's remove the old line and keep the one you added :)